tests/basic/storage_pool_test.go

```diff
@@ -4585,12 +4585,169 @@ var _ = Describe("{ResizeClusterNoQuorum}", func() {
 
 			resizeErr := waitForPoolToBeResized(expectedSize, selPool.Uuid, isjournal)
 			dash.VerifyFatal(resizeErr, nil, fmt.Sprintf("Verify pool %s on expansion using resize-disk", selPool.Uuid))
-
 		})
+		opts := make(map[string]bool)
+		opts[scheduler.OptionsWaitForResourceLeakCleanup] = true
+		ValidateAndDestroy(contexts, opts)
 
 	})
 	JustAfterEach(func() {
 		defer EndTorpedoTest()
 		AfterEachTest(contexts, testrailID, runID)
 	})
 })
+
+var _ = Describe("{PoolDelete}", func() {
+	/*1) Place pool on maintenence mode
+	2) Delete the pool
+	3) Add new pool
+	4) expand newly added pool*/
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("PoolDelete", "Initiate pool deletion", nil, 0)
+
+	})
+	var contexts []*scheduler.Context
+
+	stepLog := "Initiate pool delete, then add a new pool and expand the pool"
+
+	It(stepLog, func() {
+		log.InfoD(stepLog)
```

Comment:
This line can be removed as It(stepLog) will anyways print the same line

Reply:
log.InfoD, we are using it to push to log file and aetos dashboard